overlord/configstate/configcore/services.go

```diff
@@ -372,8 +372,20 @@ func handleServiceConfigSSHListen(dev sysconfig.Device, tr ConfGetter, opts *fsO
 
 	if opts == nil {
 		sysd := systemd.New(systemd.SystemMode, &sysdLogger{})
-		if err := sysd.ReloadOrRestart([]string{"ssh.service"}); err != nil {
-			return err
+		// From 22.10 sshd now uses socket based activation. This changes how to reload
+		// ssh configuration
```

Comment:
Super nitpick, can this be part of previous line?

Reply:
I'll move it up when doing a final revision, will just wait for another review - thank you!